docs/examples/vllm/lws.yaml

```diff
@@ -14,49 +14,60 @@ spec:
       spec:
         containers:
           - name: vllm-leader
-            # this image is build with the Dockerfile under ./build
-            image: kube-ai-registry.cn-shanghai.cr.aliyuncs.com/kube-ai/vllm:0.4.1
+            image: <image-built-from-dockerfile>
             env:
               - name: RAY_CLUSTER_SIZE
                 valueFrom:
                   fieldRef:
                     fieldPath: metadata.annotations['leaderworkerset.sigs.k8s.io/size']
+              - name: HUGGING_FACE_HUB_TOKEN
+                value: <your-hf-token>
             command:
               - sh
               - -c
               - "/vllm-workspace/ray_init.sh leader --ray_cluster_size=$RAY_CLUSTER_SIZE; 
-                 python3 -m vllm.entrypoints.openai.api_server --port 8080 --model facebook/opt-125m --swap-space 2 --tensor-parallel-size 2"
+                 python3 -m vllm.entrypoints.openai.api_server --port 8080 --model meta-llama/Meta-Llama-3.1-405B-Instruct --tensor-parallel-size 8 --pipeline_parallel_size 2"
             resources:
               limits:
-                nvidia.com/gpu: "1"
+                nvidia.com/gpu: "8"
+                memory: 1124Gi
+                ephemeral-storage: 2600Gi
+                cpu: "125"
               requests:
-                cpu: "4"
-                memory: 8Gi
-                nvidia.com/gpu: "1"
+                nvidia.com/gpu: "8"
+                memory: 1124Gi
+                ephemeral-storage: 2600Gi
+                cpu: "125"
```

Comment:
Is the requests a minimum specification? Seems huge. For example, the 405B requires less than 900GB, and we half the weights to one node. But I haven't tested this ...

Reply:
It is not a minimum specification, I can test reducing it to around 800GB since that should be around the size of 405B.

Reply:
I suggest to just set the limit and leave the request empty for both memory and gpu, for cpu do the opposite (set the requests and leave the limit empty)

Reply:
fixed